### PR TITLE
Update command line reference to align with kesm version 3.0.0

### PR DESCRIPTION
**Pull request by alch-msft**
Update command line reference to align with kesm version 3.0.0
- Fix command descriptions and parameter names
- Remove deprecated parameters 
- Add BuildIndexLocal command reference

### CODE REVIEW

File: academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md [hunk 1]
```diff
@@ -90,23 +94,43 @@ The default number of MAKES API host instances (virtual machines).
...
+
+The application settings used by the web host represented as a list of \"<ApplicationSettingJsonPath>=< ApplicationSettingValue>\" separated by ';'
+
+`--WebHostOverrideUrl`
+
+The host web app package file url reference for overriding default MAKES web host. E.g. https://mymakesstore.blob.core.windows.net/makes/2019-12-26/webhost/my-custom-makes-web-service-host.zip
+
+`--ApplicationInsightsInstrumentationKey`
+
+The Application Insights instrumentation key for sending MAKES web host logs to. E.g. 14f81de7-f9b7-4997-9cd9-d91651fe53df. By default, the logs are only stored on box as windows application events.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md:113]: hoping this is a random GUID :)

File: academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md [hunk 2]
```diff
@@ -171,37 +194,74 @@ The input entities file Url prefix.
...
+The maximum string length for all entity attributes. All strings over the maximum string length will be truncated
 
 `--WorkerCount`
 
 The number of virtual machines(workers) used to build the index. Warning, assigning a number larger than IndexPartitionCount won't result in performance gain.
 
-`--WorkerMachineSku`
+`--WorkerSku`
+
+The virtual machine(worker) sku. Use [List available VM Sizes](https://docs.microsoft.com/rest/api/compute/virtualmachines/listavailablesizes) to get the available options.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md:217]: >[List available VM Sizes](https://docs.microsoft.com/rest/api/compute/virtualmachines/listavailablesizes) [](start = 37, length = 105)

This seems like a really inconvenient way to get this information (having to make an API call)

File: academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md (hunk 1 above)
Comment (alch-msft) [reply academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md:113]: yup. it is

**Comment by alch-msft** [comment]
Moved to feature branch features/kesm-300-update

**Pull request closed without merging**